disco/util/hashmap.py

```diff
@@ -11,7 +11,7 @@ def iter(self):
         return iter(self)
 
     def items(self):
-        return six.iteritems(self)
+        return six.iteritems(super(HashMap, self))
```

Comment:
`six.iteritems()` does almost exactly that and calling `self.items()` is what causes the recursion to occur. That call still has to be delegated to the underlying dict. If you want to avoid using super:
```python
def items(self):
    return (dict.items(self) if six.PY3 else self.iteritems())
```
Is that acceptable?